### DISCUSSION TITLE
How to Use

### DISCUSSION THREAD

**Discussion by mangellotti** [sriehl/PowerSchool-Student-Map#6] [General]
Good Evening,

Are their directions on how to use this plugin?

**Comment by sriehl** [comment]
When you have a selection of students (that have geocodes), click on Student Map on the start screen. Under the student search box.

**Comment by lisakeeney13** [comment]
I have installed the latest version, but it is not grabbing all of my students. For example, if I have 132 students as my current selection and click student map, it is only mapping 40 students. I can't figure out what is going on.

**Reply by sriehl** [reply]
You'll need to be sure those missing students are geocoded. When you go to their Demographics page, I believe it shows a Latitude and Longitude by their address.

**Comment by clejeune55** [comment]
I have installed the program but it is not showing for all of my admin users
